ports/esp32/machine_pin.c

```diff
@@ -658,11 +658,19 @@ STATIC const machine_pin_irq_obj_t machine_pin_irq_object[] = {
     {{NULL}, -1}, // 30 FLASH/PSRAM
     {{NULL}, -1}, // 31 FLASH/PSRAM
     {{NULL}, -1}, // 32 FLASH/PSRAM
+    #if CONFIG_SPIRAM_MODE_OCT
+    {{NULL}, -1}, // 33 FLASH/PSRAM
+    {{NULL}, -1}, // 34 FLASH/PSRAM
+    {{NULL}, -1}, // 35 FLASH/PSRAM
+    {{NULL}, -1}, // 36 FLASH/PSRAM
+    {{NULL}, -1}, // 37 FLASH/PSRAM
+    #else
```

Comment:
the pin IRQ table towards the end of this file will probably need a similar patch applied

Reply:
Updated commits to reflect the changes for both **machine_pin_type** and **machine_pin_irq_type**.  And amended commits messages to follow guidelines.